src/components/accordion/accordion.style.ts

```diff
@@ -165,7 +182,26 @@ const StyledAccordionTitleContainer = styled.div<StyledAccordionTitleContainerPr
         : /* istanbul ignore next */ oldFocusStyling}
     }
 
+    ${variant === "subtle" &&
+    css`
+      color: var(--colorsActionMajor500);
+      padding: var(--spacing025);
+      margin-bottom: ${isExpanded && "var(--spacing200)"};
+      ${StyledAccordionIcon} {
+        color: var(--colorsActionMajor500);
+        ${iconAlign === "left" && "margin-right: var(--spacing050)"};
+      }
```

Comment:
**suggestion(non-blocking):** very minor formatting change here just to keep things consistent. 
```suggestion
      margin-bottom: ${isExpanded && "var(--spacing200)"};
      
      ${StyledAccordionIcon} {
        color: var(--colorsActionMajor500);
        ${iconAlign === "left" && "margin-right: var(--spacing050)"};
      }
```